examples/platforms/simulation/virtual_time/alarm-sim.c

```diff
@@ -105,35 +105,40 @@ void otPlatAlarmMicroStop(otInstance *aInstance)
     sIsUsRunning = false;
 }
 
-int32_t platformAlarmGetNext(void)
+uint64_t platformAlarmGetNext(void)
 {
-    int32_t remaining = INT32_MAX;
+    uint64_t remaining = INT64_MAX;
```

Comment:
OTNS treats every delay larger than 0x7fffffff as blocking forever:

https://github.com/openthread/ot-ns/blob/4a63aabc96e6c4e09eebfe450c663650bb498018/dispatcher/dispatcher.go#L319

This PR increases `remaining` to `INT64_MAX` if there is no timer pending.
But I suppose it should be fine.